components/tools/OmeroPy/test/integration/clitest/test_chgrp.py

```diff
@@ -423,5 +423,6 @@ def testNonMember(self):
 
         # try to move the image to the new group
```

Comment:
Remove ``try to`` . This implies that you are just trying to move (and possibly expecting this to fail). I would prefer the formulation ``move the image to the new group`` because this is what is being done (no try/catch or somesuch) and it also implies that a success is expected.